timeline-chart/src/components/time-graph-row-element.ts

```diff
@@ -30,6 +30,7 @@ export class TimeGraphRowElement extends TimeGraphComponent {
     ) {
         super(id, displayObject);
         this._height = _style.height || 14;
+        this._height = Math.min(this._height, _row.height -1);
```

Comment:
Can _row.height be 0? This would result in this._height being negative value. Maybe we should have clamp it to 0.

Reply:
Added the check for  _row.height equal to 0